config/dev/nodemon.json

```diff
@@ -1,7 +1,9 @@
 {
     "verbose": true,
-    "ignore": [],
+    "ignore": [
+        "./src/main/ui"
```

Comment:
`nodemon` is used to restart the entire server upon any file changes. The webpack dev server will handle changes to the UI, so we can ignore those files here.